src/gameStateService/game-state-service.ts

```diff
@@ -171,6 +184,17 @@ export class GameStateService extends PubSub {
     getStonePositions() {
         return createStonePositionsDAO(this);
     }
+    
+    levelEndSceneData({levelEndData, data}) {
+        this.levelEndData = {...levelEndData};
+        this.data = data;
+        this.isLastLevel =
+        levelEndData.currentLevel === data.levels[data.levels.length - 1].levelMeta.levelNumber;
+    }
+
+    getLevelEndSceneData() {
```

Comment:
We should share generic data in the game state, unless it is common and shared across different components.

Does this need to sit here or can it be local in the end scene component?

Reply:
To me, yes, this should be part of the game state. That way, if we need any of the data in the LevelEndScene, we can easily extract it.